generator/osm2type.cpp

```diff
@@ -939,10 +939,19 @@ void PostprocessElement(OsmElement * p, FeatureBuilderParams & params)
 
     if (!hasSuitableType)
     {
+      /// @todo Worth to add Address type for existing Entrance only when params.name is not empty.
+      /// When we will fix entrance only styles to correct draw house number.
```

Comment:
We don't need `building-address` added to `entrance*` types at all.
Housenumbers (if present) are displayed above entrance's icons just like with any other POI.

Reply:
and actually ATM presence of `building-address` hides entrance's icons in vehicle style, because `building-address` has a higher prio there (we wanted to prioritize housenumbers in nav mode compared to other micromapping)

Reply:
Ok, did you test it locally? Remove adding for "entrance" and see how it is drawn. Because all our maps generation was with this change.

Reply:
https://github.com/organicmaps/organicmaps/pull/6537/commits/b3c4104bbdec795da24773da1bedf22b472a47b4

Reply:
I double-checked - it works fine!